### ISSUE TITLE
URL support not working on Mac OS X

### ISSUE THREAD

**Issue by bwklein** [asciidoctor/atom-asciidoc-image-helper#7]
Report from @chuckf originally on https://github.com/bwklein/asciidoc-image-helper/issues/20

Using Atom 1.7.3 and asciidoc-image-helper 0.9.2, when I paste a path to an image file that path pastes in and does not convert to an image:: link as expected.

The test path I have used is:

```
/Users/chuckf/Downloads/splunk.png
/Users/chuckf/Downloads/splunk.jpg
```

Expected result is

```
image::splunk.png
image::splunk.jpg
```

Actual result:

```
/Users/chuckf/Downloads/splunk.png
/Users/chuckf/Downloads/splunk.jpg
```

NOTE: When pasting similar paths on a Fedora box, same versions of the software, the URLs are converted to `image:: links` as expected.

I don't get a success or error message popup.

I opened the console window with <kbd>cmd-alt-i</kbd> (for future reference) and did not see any errors occur when pasting in the link.

I typed the paths into a text editor, HTML page, and ran the command `ls /Users/chuckf/Downloads/splunk.png` into the native terminal and iterm2. I then selected the text and copied using <kbd>cmd-c</kbd> and pasted into Atom using <kbd>cmd-v</kbd>.

**Comment by nicorikken** [comment]
Just tried it out and can confirm the issue. This functionality did work previously. Small caveat: I just restructured my package install, which might have been messing things up. (but of course no way to verify 😉  )

**Comment by bwklein** [comment]
I think that it might be how we are handling the string in Line 88.
https://github.com/asciidoctor/atom-asciidoc-image-helper/blob/master/lib/main.coffee#L88

**Comment by nicorikken** [comment]
Then this commit would have messed things up? https://github.com/asciidoctor/atom-asciidoc-image-helper/commit/c1e4fe7ee166d0b6990b6f4db3395586c2a73894

**Comment by bwklein** [comment]
Should be fixed in #13
